src/main/java/net/discordjug/javabot/listener/filter/MessageModificationStatus.java

```diff
@@ -0,0 +1,24 @@
+package net.discordjug.javabot.listener.filter;
+
+/**
+ * This enum describes the result of an {@link MessageFilter} execution and what should be done with the message.
```

Comment:
I think it should be "_a_ MessageFilter execution", not "an MessageFilter execution".